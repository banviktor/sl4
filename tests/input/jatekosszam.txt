randomization off
questions off
start_game 3
2 1
3 3
5 4
list_player_robots
exit
