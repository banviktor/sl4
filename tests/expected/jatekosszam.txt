Véletlenszerű események kikapcsolva.
Kérdezés kikapcsolva.
Pálya betöltve.
Új játék elkezdve 3 játékossal.
Ez a(z) 1. kör.
A Piros Robot következik.
Játékosrobotok listája:
	Piros: 2.0, 1.0
	Sárga: 3.0, 3.0
	Zöld: 5.0, 4.0
